spec/api/commentable_spec.rb

```diff
@@ -32,6 +32,11 @@ def thread_result(commentable_id, params={})
         threads.index{|c| c["body"] == "can anyone help me?"}.should_not be_nil
         threads.index{|c| c["body"] == "it is unsolvable"}.should_not be_nil
       end
+      it "returns standalone threads if explicitly requested" do
+        threads = thread_result "question_1", context: "standalone"
```

Comment:
it doesn't look like the ability to fetch a single thread by its id (regardless of its context) has changed, if that's what you mean.